homeassistant/components/light/__init__.py

```diff
@@ -239,6 +241,8 @@ def get_supported_color_modes(hass: HomeAssistant, entity_id: str) -> set[str] |
 VALID_BRIGHTNESS_PCT = vol.All(vol.Coerce(float), vol.Range(min=0, max=100))
 VALID_BRIGHTNESS_STEP = vol.All(vol.Coerce(int), vol.Clamp(min=-255, max=255))
 VALID_BRIGHTNESS_STEP_PCT = vol.All(vol.Coerce(float), vol.Clamp(min=-100, max=100))
+VALID_BRIGHTNESS_MIN = vol.All(vol.Coerce(int), vol.Clamp(min=0, max=255))
+VALID_BRIGHTNESS_MIN_PCT = vol.All(vol.Coerce(float), vol.Clamp(min=0, max=100))
```

Comment:
This is a change to our main entity models/functionality, which requires an architectural discussion and approval before implementation.

I could not find an architectural discussion in our architecture repository about this. Could you point me to it?

../Frenck

Reply:
Apologies, created a discussion here: https://github.com/home-assistant/architecture/discussions/819